internal/pkg/admission/policy-server-deployment.go

```diff
@@ -146,7 +145,6 @@ func (r *Reconciler) updatePolicyServerDeployment(ctx context.Context, policySer
 		patch := originalDeployment.DeepCopy()
 		patch.Spec.Replicas = newDeployment.Spec.Replicas
 		patch.Spec.Template = newDeployment.Spec.Template
```

Comment:
should we add the constants.PolicyServerDeploymentConfigVersionAnnotation] annotation to the patch?